build.sbt

```diff
@@ -11,5 +11,16 @@ lazy val root = (project in file("."))
 
```

Comment:
Should add the the assembly commands to the build file.

See here for a recent example for a lambda: https://github.com/nationalarchives/tdr-export-status-update/blob/81ac224d2f34046b6b57dcf7f7ba9944ddf84a80/build.sbt